system-int-test/src/cmr/system_int_test/utils/generic_util.clj

```diff
@@ -0,0 +1,91 @@
+(ns cmr.system-int-test.utils.generic-util
+  (:require
+   [cheshire.core :as json]
+   [clj-http.client :as client]
+   [clojure.data :as d]
+   [clojure.data.xml :as x]
+   [clojure.string :as str]
+   [clojure.test :refer :all]
+   [cmr.acl.core :as acl]
+   [cmr.common-app.config :as common-config]
+   [cmr.common-app.test.side-api :as side]
+   [cmr.common.mime-types :as mt]
+   [cmr.common.util :as util]
+   [cmr.common.xml :as cx]
+   [cmr.ingest.config :as ingest-config]
+   [cmr.mock-echo.client.echo-util :as echo-util]
+   [cmr.system-int-test.data2.provider-holdings :as ph]
+   [cmr.system-int-test.system :as sys]
+   [cmr.system-int-test.utils.dev-system-util :as dev-sys-util]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.metadata-db-util :as mdb]
+   [cmr.system-int-test.utils.url-helper :as url-helper]
+   [cmr.transmit.access-control :as ac]
+   [cmr.transmit.config :as transmit-config]
+   [cmr.umm-spec.versioning :as umm-versioning]
+   [cmr.umm.echo10.echo10-collection :as c]
+   [cmr.umm.echo10.echo10-core :as echo10]
+   [cmr.umm.echo10.granule :as g])
+  (:import
+   [java.util UUID]))
+
+(def grid-good
+  {:MetadataSpecification {:URL "https://cdn.earthdata.nasa.gov/generic/grid/v0.0.1"
```

Comment:
if you have the the schemas in system-int-test, you can replace this with the following:

`(def grid-good (-> "schemas/grid/v0.0.1/metadata.json"
                    (jio/resource)
                    (slurp)
                    (json/parse-string true)))`